apps/aecore/src/aec_headers.erl

```diff
@@ -785,7 +785,7 @@ validate_pow(#key_header{nonce        = Nonce,
 validate_micro_block_cycle_time(Header, _Protocol) ->
     Time = time_in_msecs(Header),
     PrevHash = prev_hash(Header),
-    case aec_chain:get_header(PrevHash) of
+    case aec_chain:get_header(PrevHash) of %% TODO: Investigate why dirty read crashes tests with {no_exists}
```

Comment:
Could the crash be because the header of `PrevHash` resides in the transaction store, not yet committed?